src/app/create-task/create-task.component.html

```diff
@@ -0,0 +1,63 @@
+<form [formGroup]="currentTask" (ngSubmit)="onSubmit()">
+
+  <div class="form-layout">
+    <div>
+      <label for="title">Title:</label>
+      <input id="title" type="text" formControlName="title" required>
+      <div *ngIf="title?.invalid && title?.touched" class="error-message">
```

Comment:
you could create a separate error-message component where you pass the form field errors and display the relevant ones